RevenueCatUI/CustomerCenter/Abstractions/CustomerCenterPurchasesType.swift

```diff
@@ -44,4 +50,15 @@ protocol CustomerCenterPurchasesType: Sendable {
 
     func restorePurchases() async throws -> CustomerInfo
 
+    // MARK: - Subscription Management
+
+    #if os(iOS) || os(macOS) || os(visionOS)
+    @Sendable
+    func showManageSubscriptions() async throws
+    #endif
+
+    #if os(iOS) || os(visionOS)
+    @Sendable
+    func beginRefundRequest(forProduct productID: String) async throws -> RefundRequestStatus
+    #endif
```

Comment:
These compiler conditions are exactly the same as those in the corresponding methods in `Purchases.swift`